docs/modules/release-notes/pages/5-3-0.adoc

```diff
@@ -14,7 +14,7 @@ without the need of a Kafka cluster.
 * Renamed the `DataLinkFactory` interface as `DataConnection`.
 https://github.com/hazelcast/hazelcast/pull/24224[#24224]
 * Changed the default cloud coordinator URL from `coordinator.hazelcast.cloud` to `api.viridian.hazelcast.com`.
-The default configuration now connects to https://viridian.hazelcast.com/sign-in?next=/[Hazelcast Viridian Cloud^] instead of Hazelcast Cloud.
+The default configuration now connects to https://viridian.hazelcast.com/sign-in?next=/?utm_source=docs-website[Hazelcast Viridian Cloud^] instead of Hazelcast Cloud.
```

Comment:
```suggestion
The default configuration now connects to https://viridian.hazelcast.com/sign-in?utm_source=docs-website[Hazelcast Viridian Cloud^] instead of Hazelcast Cloud.
```